### PR TITLE
updated guidelines

### PR DESCRIPTION
**Pull request by aharonha**

### CODE REVIEW

**Review by artem-v — APPROVED** [review]

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+You can use [google-java-format](https://github.com/google/google-java-format).
+There are plugins for [IntelliJ](https://github.com/google/google-java-format#intellij) and [Eclipse](https://github.com/google/google-java-format#eclipse) 
+
+
+## IntelliJ Users (that doesn't want to install the plugin for some reason) should:
+0. Download [this code style configuration](https://github.com/google/styleguide/blob/gh-pages/intellij-java-google-style.xml)
+Apply the configuration:
+
+## Eclipse Users (that doesn't want to install the plugin for some reason) should:
```
Comment (ronenhamias) [inline DEVELOPMENT.md:26]: missing instruction regards import order in eclipse

Comment (aharonha) [reply DEVELOPMENT.md:26]: I believe that an empty one would suffice.
I didn't test it though.

Comment (ronenhamias) [reply DEVELOPMENT.md:26]: ok so i will wait for something with after you have clear guideline regards this issue

Comment (aharonha) [reply DEVELOPMENT.md:26]: As https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing suggests, I believe a file like this: 
```
0=\#
```
is good enough

Comment (aharonha) [reply DEVELOPMENT.md:26]: That's not the default one, it's an empty one.

**Comment by dmytro-lazebnyi** [comment]
What about line length?
100 as defined by google style or change to 120?

**Comment by ronenhamias** [comment]
i like 120 better

**Comment by dmytro-lazebnyi** [comment]
@ronenhamias @aharonha 
If we want to use `120`, then it is needed to remove `google-java-format` plugin from instructions, because as far as I understand this plugin use built-in code formatter and doesn't allow to change properties. 
So, if you use `google-java-format` plugin you can't change line length from `100` to `120`

**Comment by ronenhamias** [comment]
so maybe better to accept 100 ?

**Comment by snripa** [comment]
I'm for 100 line length

**Comment by segabriel** [comment]
OK if we will have no problem with 100 and without extended efforts - I agree with 100 line length

**Comment by snripa** [comment]
so the majority is for 100 as I understood

**Review by snripa — COMMENTED** [review]
100 line length is fine

**Review by snripa — APPROVED** [review]

**Review by segabriel — APPROVED** [review]

**Pull request merged**